bleak/backends/__init__.py

```diff
@@ -1,5 +1,58 @@
-# -*- coding: utf-8 -*-
-# Created on 2017-11-19 by hbldh <henrik.blidh@nedomkull.com>
-"""
-__init__.py
-"""
+import enum
+import os
+import platform
+import sys
+
+from bleak.exc import BleakError
+
+
+class BleakBackend(str, enum.Enum):
```

Comment:
```suggestion
class BleakBackend(enum.StrEnum):
```

Unless I'm missing something?

Reply:
> Added in version 3.11.

Oops, I am missing something. :facepalm: